modules/ossm-cr-gateway.adoc

```diff
@@ -12,61 +12,63 @@ Here is an example that illustrates the Istio gateway parameters for the `Servic
 [source,yaml]
 ----
   gateways:
-       istio-egressgateway:
-         autoscaleEnabled: false
-         autoscaleMin: 1
-         autoscaleMax: 5
-       istio-ingressgateway:
-         autoscaleEnabled: false
-         autoscaleMin: 1
-         autoscaleMax: 5
-         ior_enabled: true
+    egress:
+      enabled: true
+      runtime:
+        deployment:
+          autoScaling:
+            enabled: true
+            maxReplicas: 5
+            minReplicas: 1
+    enabled: true
+    ingress:
+      enabled: true
+      runtime:
+        deployment:
+          autoScaling:
+            enabled: true
+            maxReplicas: 5
+            minReplicas: 1
 ----
 
 
 .Istio Gateway parameters
 |===
 |Type |Parameter |Description |Values |Default value
 
-|`istio-egressgateway`
-|`autoscaleEnabled`
+|`gateway.egress`
```

Comment:
Since "Type" can appear in the YAML file, we should probably be careful about using it in table headings describing YAML configuration to avoid confusion.   (And yes, we should probably go back and clean it up in the other tables) I don't see a "Type" in my YAML file anywhere before the gateway configuration.
Instead from the YAML I've got it looks like it these values should not be split into two columns but noted as:
`gateways.egress.runtime.deployment.autoScaling.enabled`
`gateways.egress.runtime.deployment.autoScaling.minReplicas`
`gateways.egress.runtime.deployment.autoScaling.maxReplicas`  etc.
(@rcernich can you verify should gateways be plural?)

Reply:
Yes